### DISCUSSION TITLE
MySQL fulltext search , elastic search 에서 Query DSL 사용한 검색 성능 비교

### DISCUSSION THREAD

**Discussion by dev-Rhea** [100-hours-a-week/discussions#13] [Deepdive]
검색 기능을 구현하며 작성한 쿼리에 따른, 검색 기능의 성능 향상에 대한 글을 작성할 예정입니다.

**Comment by startupcodekr** [comment]
현재 개발하고 있는 프로젝트와 관련성이 보이지 않습니다

**Reply by dev-Rhea** [reply]
1. MySQL 에서의 Query DSL 을 사용한 검색 
2. elasticsearch에서 Query DSL 을 사용한 검색 

두 가지를 비교하려 했는데, 범위가 많은가요?

**Reply by startupcodekr** [reply]
이 주제를 프로젝트 관점에서 왜 해야하는지가 필요해요.

**Reply by dev-Rhea** [reply]
제가 프로젝트에서 검색 기능 구현을 맡게 되어서 이 주제로 정했습니다! 
저희 프로젝트는 특가 정보 서비스로, 다양한 상품을 사용자에게 제공하기 위해 준비 중이며 다양한 상품이 존재하는 만큼 사용자가 원하는 상품을 쉽고 빠르게 찾기 위해서 검색 기능을 넣게 되었습니다.
